frontend/smart-kickers-game/src/App.js

```diff
@@ -31,13 +43,36 @@ function App() {
     });
   }
 
+  function handleEndGame() {
+    setToggleGameScore(!toggleGameScore);
+  }
+
   return (
     <>
       <h1>Smart Kickers</h1>
-      <GameResults blueScore={blueScore} whiteScore={whiteScore} />
-      <center>
-        <Button onClick={() => handleResetGame()}>Reset game</Button>
-      </center>
+      {toggleGameScore === false ? (
```

Comment:
Have you considered using a router for such a usecase?

Reply:
We thought about it but we decided that this will be easier for us and the user can not comeback to the live game view as we intendent.